dts/bindings/lora/semtech,sx126x-base.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2019 Manivannan Sadhasivam
+# Copyright (c) 2020 Andreas Sandberg
+# SPDX-License-Identifier: Apache-2.0
+
+include: spi-device.yaml
+
+properties:
+    reset-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        GPIO connected to the sensor NRST signal.
+
+        This signal is open-drain, active-low as interpreted by the
+        modem.
+
+    busy-gpios:
+      type: phandle-array
+      required: true
+      description: |
+        GPIO connected to the sensor BUSY signal.
+
+    irq-gpios:
```

Comment:
I would have a slight preference for dio1-gpios, as DIO1 is what we see usually on schematics. `busy-gpios` can be configured as interrupt as well in the future, so we could get rid of busy (with k_sleep) loop some day.

Reply:
That makes sense. I have updated the bindings.

Reply:
Bindings are updated, but not in the right commit :)

Reply:
Sigh, I thought I made sure to apply the fixups to the right commits. I’ll post an update tonight.